src/ducks/balance/Balance.jsx

```diff
@@ -136,6 +143,54 @@ class Balance extends PureComponent {
     })
   }
 
+  startRealtime(type, callback) {
+    if (!this.realtime[type]) {
+      const cozyClient = this.props.client
+      const url = cozyClient.client.uri
+      const token = cozyClient.client.token.token
+      this.realtime[type] = new CozyRealtime({ url, token })
+      this.realtime[type].subscribe({ type }, 'created', callback)
+    }
+  }
+
+  stopRealtime(type, callback) {
+    if (this.realtime[type]) {
+      this.realtime[type].unsubscribe({ type }, 'created', callback)
+      delete this.realtime[type]
```

Comment:
Is the websocket closed when every unsubscribe has been made ?

Reply:
not at the moment

Reply:
I'm writing a todo list with this issue on https://github.com/cozy/cozy-libs/pull/355